source/includes/openstack/_securityGroups.md

```diff
@@ -6,7 +6,9 @@ Security groups manage network access to instances.
 
 ```shell
 curl -H "MC-Api-Key: your_api_key" \
+
     "https://api.your.cloudmc/v1/services/compute-os/devel/securitygroups"
+
```

Comment:
Think i had a merge conflict, must have accidently added them when i was trying to resolve it. Ill fix it.